packages/stylelint-config/package.json

```diff
@@ -22,10 +22,10 @@
     "node": ">=10"
   },
   "dependencies": {
-    "stylelint-config-standard": "^20.0.0",
+    "postcss-scss": "^4.0.3",
```

Comment:
I do not understand this change. Reading https://stylelint.io/migration-guide/to-14/ suggests that the appropriate dependency should be `stylelint-config-standard-scss`.

From the docs seems to be `postcss-scss` ok for custom syntax, but what about shared config we were using?

Reply:
The thing is we did _not_ use it. It was only installed, but not used.

Stylelint migration guide suggests installing `stylelint-config-standard-scss` which covers all necessary dependencies for reading SCSS, but it also adds some SCSS related rules and extends `stylelint-config-standard` which is not what we (currently) want. The scope of this change is only to provide compatibility with Sass again. 

However, we may get back to community configs during #24.